<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>812</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="ArduinoConfiguration_groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>471</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Arduino Configuration</string>
    </property>
    <widget class="QComboBox" name="PortRate_comboBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>300</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>28800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>230400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>250000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>500000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1000000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2000000</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="OpenPort_pushButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Open the Port</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ClosePort_pushButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>90</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Close the port</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Port Rate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AutoScan_pushButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Auto scan</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Port number</string>
     </property>
    </widget>
    <widget class="QComboBox" name="PortNumber_comboBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="DisplayFunction_groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>961</width>
      <height>591</height>
     </rect>
    </property>
    <property name="title">
     <string>Display function</string>
    </property>
    <widget class="QGroupBox" name="ArbitraryDisplayMode_groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>921</width>
       <height>311</height>
      </rect>
     </property>
     <property name="title">
      <string>Arbitrary Display Mode</string>
     </property>
     <widget class="QGroupBox" name="SmallRing_groupBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>421</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>Small Ring</string>
      </property>
      <widget class="QPushButton" name="SmallRingApply_pushButton">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>230</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox1">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>50</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox2">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>70</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox4">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>150</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox5">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>180</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox6">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>150</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox7">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>110</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox8">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox9">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>110</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
      <widget class="QPushButton" name="reset_small_pushButton">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox3">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>110</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="LargeRing_groupBox">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>40</y>
        <width>421</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>Large Ring</string>
      </property>
      <widget class="QPushButton" name="LargeRingApply_pushButton">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>230</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_1">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>40</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>60</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_4">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>90</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_5">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>120</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_6">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>150</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_7">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>180</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_8">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>200</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_9">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>210</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_10">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>200</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>10</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_11">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>180</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>11</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_12">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>150</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>12</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_13">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>120</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>13</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_14">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>90</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>14</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_15">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>15</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_16">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>16</string>
       </property>
      </widget>
      <widget class="QPushButton" name="reset_large_pushButton">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QGroupBox" name="LoopDisplayMode_groupBox">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>40</y>
       <width>371</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string>Loop Display Mode</string>
     </property>
     <widget class="QPushButton" name="Short_loop_pushButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Short loop time</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Medium_loop_pushButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Medium loop time</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Long_loop_pushButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>160</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Long loop time</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="DefaultDisplayMode_groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>451</width>
       <height>241</height>
      </rect>
     </property>
     <property name="title">
      <string>Default Display Mode</string>
     </property>
     <widget class="QPushButton" name="OddBlink_pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Odd index LED blink</string>
      </property>
     </widget>
     <widget class="QPushButton" name="EvenBlink_pushButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>160</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Even index LED blink</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LeftOn_pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Left LED on</string>
      </property>
     </widget>
     <widget class="QPushButton" name="RightOn_pushButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>120</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Right LED on</string>
      </property>
     </widget>
     <widget class="QPushButton" name="IndividualBlink_pushButton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>200</y>
        <width>191</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Individual Blink</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AllLEDOn_pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>All LED on</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AllLEDOff_pushButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>80</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>All LED off</string>
      </property>
     </widget>
     <widget class="QPushButton" name="White_pushButton">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>White</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Red_pushButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Red</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Color Wipe</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Green_pushButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Green</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Blue_pushButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Blue</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>993</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
